graphite/graphite.go

```diff
@@ -100,6 +101,15 @@ func readTraceback(resp *http.Response) (*[]string, error) {
 // DefaultClient is the default HTTP client for requests.
 var DefaultClient = &http.Client{
 	Timeout: time.Minute,
+	Transport: &http.Transport{
+		MaxIdleConnsPerHost: 0,
```

Comment:
i don't really have time any time soon to try to make it reproducable and verify the fix.
my admittedly crummy plan is to just set http.DefaultMaxIdleConnsPerHost to 0 , run that in prod, and check if errors disappear.